deploy/docker/seqr/Dockerfile

```diff
@@ -55,9 +64,9 @@ RUN apt-get update \
     && apt-get install -y \
         nodejs
 
-# install database clients for debugging
-RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' >> /etc/apt/sources.list.d/postgresql.list \
-    && wget --no-check-certificate -q https://www.postgresql.org/media/keys/ACCC4CF8.asc -O- | apt-key add - \
+# install database clients for debugging (https://www.postgresql.org/download/linux/debian/)
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list \
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5815111)